Add the missing package version and expose the BWA executable setting used by
the read-mapping code.

Author: Emik Lin (HKUMed CPOS)
--- a/setup.py
+++ b/setup.py
@@ -1,6 +1,7 @@
 from setuptools import setup
 
 setup(name='metaMIC',
+      version='20220221',
       description='metaMIC: Reference-free Misassembly Identification and Correction of metagenomic assemblies',
       long_description_content_type = 'text/markdown',
       classifiers=[
--- a/metaMIC/metaMIC.py
+++ b/metaMIC/metaMIC.py
@@ -174,6 +174,13 @@ def main():
         required=False,
         help='Minimum contig length [default: 5000bp]')
 
+    extract_feature.add_argument(
+        "--bwa",
+        dest="bwa",
+        default='bwa',
+        required=False,
+        help="path to BWA")
+
     extract_feature.add_argument(
         "--samtools",
         dest="samtools",
